sdk/template/template/recordings/browsers/aad_configurationclient_functional_tests/recording_predetermined_setting_has_expected_value.json

```diff
@@ -17,8 +17,8 @@
         "Sec-Fetch-Mode": "cors",
         "Sec-Fetch-Site": "same-site",
         "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/93.0.4577.0 Safari/537.36",
-        "x-ms-client-request-id": "0a240cc2-fffd-45da-9a4c-fe8cebdff715",
-        "x-ms-useragent": "azsdk-js-template-template/1.0.11-beta.1 core-http/2.2.3 OS/Linuxx86_64"
+        "x-ms-client-request-id": "60acdfbf-20db-4571-b5a4-fa921627eebc",
+        "x-ms-useragent": "azsdk-js-template-template/1.0.11-beta.1 azsdk-js-template/1.0.11-beta.1 core-rest-pipeline/1.4.1 OS/Linuxx86_64"
```

Comment:
Why do we need the `"...template/1.0.11-beta.1"` part twice?

Reply:
The new generated client context now also handles user agent options so causing duplicates. thus my comment to [remove it in the convenience layer](https://github.com/Azure/azure-sdk-for-js/pull/19748#discussion_r790075073)

Reply:
Yes, I just removed that. Maybe the recordings would need to be re-recorded too